docs/semgrep-cloud-platform/user-management.md

```diff
@@ -155,6 +151,18 @@ To **change a user's role**:
 You cannot change your own role.
 :::
 
+## Setting a default role
+
+Organizations start with a default role of `admin`
+
+To change this:
```

Comment:
```suggestion
To change this, perform the following steps:
```
This is a minor style thing, but we use full sentences when starting a procedure. (See: [Introductory sentences](https://developers.google.com/style/procedures#introductory-sentences))